src/visualizer/codeinformation/codetemplate/Selector.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import './styles.css';
+
+/**
+ * Selector component which allows the users to select the code corresponding to the programming
+ * language.
+ *
+ * @component
+ * @category CodeTemplate
+ * @returns {JSX.Element} Selector component.
+ */
+const Selector = ({ selected, setSelected }) => {
+  /**
+   * Select button for users to choose the programming language.
+   *
+   * @param language Programming language.
+   * @returns {JSX.Element} Select button.
```

Comment:
Maybe could write it as @param language. Programming language.
That's what I did for most of my params, it makes it slightly more readable